invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchDropdowns.js

```diff
@@ -62,7 +62,7 @@ export class DropdownSort extends Component {
         trigger={
           <span className="flex align-items-center">
             {this.getCurrentlySelectedOption(parsedOptions).text}
-            <Icon name="dropdown" role="option" />
+            <Icon name="dropdown" />
```

Comment:
shouldn't the role be set on the Dropdown instead?

Reply:
The Icon was inside a parent element with role listbox so it required the child elements to have the role option, which wouldn't work with icon, I added it but with screen reader it won't make sense, so removed it later